neurons/miner.py

```diff
@@ -62,17 +62,18 @@ def __init__(self, config=None):
     def load_config(self):
         # default values
         config = configparser.ConfigParser()
-        config.read('config.properties')
+        config.read('miner.properties')
         
         # Load symbols, model path, and currency from DEFAULT section
         if config.has_section('DEFAULT'):
-            symbols = config.get('DEFAULT', 'symbols', fallback='tao').split(',')
+            symbols = config.get('DEFAULT', 'symbols', fallback='tao').split(',') # Default to 'tao'
             model_path = config.get('DEFAULT', 'model_path', fallback=None)
-            currency = config.get('DEFAULT', 'currency', fallback='usd')  # Default to 'usd' if not specified
+            currency = config.get('DEFAULT', 'currency', fallback='usd')  # Default to 'usd'
+            intervals = config.get('DEFAULT', 'intervals', fallback='5m').split(',')  # Default to '5m'
             
-            return [symbol.strip() for symbol in symbols], model_path, currency  # Strip whitespace from symbols
+            return [symbol.strip() for symbol in symbols], model_path, currency, intervals  # Strip whitespace from symbols
         
-        return [], None, 'usd'  # Return empty list, None for model path, and default currency
+        return [], None, 'usd', '5m'  # Return empty list, None for model path, default currency and intervals
```

Comment:
is the 5m the time frame interval? should it be hardcoded?

Reply:
It is just a default value but the code will fail if this case happens as we cannot continue without a model.